.cirrus.yml

```diff
@@ -8,27 +8,44 @@ env:  # Global defaults
   CCACHE_DIR: "/tmp/ccache_dir"
   CCACHE_NOHASHDIR: "1"  # Debug info might contain a stale path if the build dir changes, but this is fine
 
+# A self-hosted machine(s) can be used via Cirrus CI. It can be configured with
+# multiple users to run tasks in parallel. No sudo permission is required.
+#
 # https://cirrus-ci.org/guide/persistent-workers/
 #
-# It is possible to select a specific persistent worker by label. Refer to the
+# Generally, a persistent worker must run Ubuntu 23.04+ or Debian 12+.
+#
+# The following specific types should exist, with the following requirements:
+# - small: For an x86_64 machine, recommended to have 2 CPUs and 8 GB of memory.
+# - medium: For an x86_64 machine, recommended to have 4 CPUs and 16 GB of memory.
+# - arm64: For an aarch64 machine, recommended to have 2 CPUs and 8 GB of memory.
+#
+# CI jobs for the latter configuration can be run on x86_64 hardware
+# by installing qemu-user-static, which may work out of the box with
+# podman or docker.
```

Comment:
> which may work out of the box with podman or docker.

Please don't add vaugeries like this to docs. If you have a specific issue that you're running into, please open an issue, otherwise things should be assumed to be working (otherwise they should be fixed, rather than adding docs claiming they might not work).

Reply:
I don't know if it works out of the box for every distro out there. It does for Ubuntu 24.04 when used with podman, at least after a reboot. I haven't tested with docker nor on any other distro. The documentation for https://github.com/multiarch/qemu-user-static says you need to run `sudo podman run --rm --privileged multiarch/qemu-user-static --reset -p yes` first, but this is no longer true.

What I'm trying to convey with "may work" is: it might work with just `sudo apt install qemu-user-static`, even though the projects own README says it doesn't. I can change it to "works", but that might be wrong - hard to know without trying every permutation out there.

Reply:
This uncertainty is also part of the reason #29274 adds `NO_ARM` as an option.

Reply:
> I don't know if it works out of the box for every distro out there

That's fine. Just assume it does until we know otherwise.

Reply:
Ok, changed to "works" and added a link that explains what this does.